command/metrics.go

```diff
@@ -88,14 +99,31 @@ func (c *OperatorMetricsCommand) Run(args []string) int {
 		Params: params,
 	}
 
-	bs, err := client.Operator().Metrics(query)
-	if err != nil {
-		c.Ui.Error(fmt.Sprintf("Error getting metrics: %v", err))
-		return 1
+	if json || len(tmpl) > 0 {
```

Comment:
for a string I'd just check `tmpl != ""`

Reply:
This was copied verbatim from the other commands that have -json and -t flags.  Leave as is or change?

Reply:
No need to change, just a nit pick